tutor/src/models/course/scores/task-result.js

```diff
@@ -25,6 +25,7 @@ export default class TaskResult extends BaseModel {
   @field correct_on_time_step_count;
   @field correct_accepted_late_step_count;
   @field({ type: 'date' }) due_at;
+  @field({ type: 'date' }) last_worked_at;
   @field exercise_count;
```

Comment:
This is the bug we're fixing.  Since the model didn't have a field it ignored the value from the BE